src/app/fetcher/htmlOnlyFetcher.js

```diff
@@ -3,7 +3,7 @@ import HttpProxyAgent from 'http-proxy-agent';
 import HttpsProxyAgent from 'https-proxy-agent';
 import nodeFetch, { AbortError } from 'node-fetch';
 
-import { InaccessibleContentError } from '../errors.js';
+import { FetchError } from './errors.js';
```

Comment:
This name is very technical. What about `InaccessibleDocumentError`? 🙂

Reply:
I believe t's clearer if it's a `FetchError`